gnomecast.py

```diff
@@ -641,6 +658,11 @@ def on_cast_combo_changed(self, combo):
       else:
           entry = combo.get_child()
 
+  def on_volume_changed(self, widget):
```

Comment:
i'm prob. going to add a `@throttle()` here so we're not pushing too many calls.